.github/workflows/scripts/build-images.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+OUTPUT="$1"
+
+mkdir "$OUTPUT"
+
+# Build everything except mimir (run by build-mimir job) and build image (not managed by CI).
```

Comment:
Where is "mimir" excluded from building here? I'm missing it.

Reply:
Wrong copy&pasted comment.